internal/bminventory/inventory.go

```diff
@@ -798,9 +798,7 @@ func (b *bareMetalInventory) getNewClusterCPUArchitecture(newClusterParams *mode
 		return common.DefaultCPUArchitecture, nil
 	}
 
-	if !swag.BoolValue(newClusterParams.UserManagedNetworking) && !featuresupport.IsFeatureSupported(swag.StringValue(newClusterParams.OpenshiftVersion),
-		models.FeatureSupportLevelFeaturesItems0FeatureIDARM64ARCHITECTUREWITHCLUSTERMANAGEDNETWORKING) {
-
+	if !swag.BoolValue(newClusterParams.UserManagedNetworking) && !featuresupport.IsArchitectureSupported(models.ArchitectureSupportLevelIDARM64ARCHITECTURE, *newClusterParams.OpenshiftVersion) {
```

Comment:
not sure that you didn't change the behavior here - we should test it based on the old code 
4.10 - ARM - supported 
4.10 - ARM - with cluster manage network - unsupported 
4.11 - ARM - with cluster manage network - supported